file-mon/src/main/java/com/wipro/ats/bdre/filemon/QueuedFileUploader.java

```diff
@@ -40,6 +39,8 @@
     private static SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss");
     private static Configuration config = new Configuration();
 
+    private QueuedFileUploader(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)